tutorials/repl/replTestOut.txt

```diff
@@ -8,7 +8,7 @@ false
 true
 -40
 104
-runtime error: error: Variable kettle::temperature is not set
+runtime error: error: Variable temperature is not set
```

Comment:
I changed this and the markdown by hand because setting up a go env to build `lmt` didn't fit my schedule for this week :stuck_out_tongue_closed_eyes:. I hope it doesn't break automatic generation.